### PR TITLE
format: use shfmt to format sh files.

### PR DESCRIPTION
**Pull request by jcvenegas**
Use shfmt to format all the sh file.

https://github.com/mvdan/sh

### CODE REVIEW

File: kernel/build-kernel.sh [hunk 1]
```diff
@@ -273,7 +273,7 @@ main() {
 		esac
 	done
 
-	shift $(($OPTIND - 1))
+	shift $((OPTIND - 1))
```
Comment (ydjainopensource) [inline kernel/build-kernel.sh]: See this [here](https://unix.stackexchange.com/a/214151).

Do you wish to change it to  ```shift "$((OPTIND-1))"```?

**Comment by jodh-intel** [comment]
lgtm!

Nice one @jcvenegas!

I'm afraid you've been caught by your own trap though :smile::

```
Kernel directory has changes check kernel/kata_config_version changed
ERROR: Please bump version in kernel/kata_config_version
```

If you get `shfmt` run on all the repos, it probably makes sense to integrate it into the CI in:

- https://github.com/kata-containers/tests/blob/master/.ci/static-checks.sh

**Comment by grahamwhaley** [comment]
If we are going to do that, should we also consider https://github.com/koalaman/shellcheck ? @jodh-intel - is that the one we discussed before (or is there another lint-for-shell you know of?).

**Comment by jodh-intel** [comment]
Sounds good.

File: kernel/build-kernel.sh (hunk 1 above)
Comment (jcvenegas) [reply kernel/build-kernel.sh]: @ydjainopensource yeah :) , I think next PR will be to quote a lot of the variables we use :).

**Comment by jcvenegas** [comment]
@jodh-intel  @grahamwhaley  yes that would be great,  we have a lot of bash code in different repos hope we can (someday) in all repos :D .

**Pull request merged**